tests/ytt/uaa/uaa-values.yml

```diff
@@ -8,8 +8,8 @@ data:
   uaa.yml:
 
 internal_certificate:
-  key:
-  crt:
+  key: internal-cert-key
```

Comment:
These looks related/unrelated to me.  How does moving where base64 encoding mean that we now need to pass in a value for key and crt here?